client/src/components/HomePage.js

```diff
@@ -55,7 +37,7 @@ const HomePage = (props) => {
                 <div class="headerText">
                     <h2 class="page header" align="center">Welcome to the most Radical Review Roadblock</h2>
 
-                    <p align="center" class="text">Hey there fellow x-gamer! Have you ever been to restaurant and thought to yourself "man this is totatally not tubular man!" Well we share your pain dude guy man. We want to bring to you a place where you can totally check out the best places and put the poopy buttholes in their place!</p>
+                    <p align="center" class="text">Hey there fellow x-gamer! Have you ever been to restaurant and thought to yourself "woah this is totatally not tubular!" Well we share your pain. We want to bring to you a place where you can totally check out the best places and put the not so great restaurants in their place!</p>
```

Comment:
Have you ever been to "a" restaurant. Missing an a there. "Well, we share your pain." I think a comma would sound better here